docs/core-concepts/core-sdk/cli/dotnet-new.md

```diff
@@ -0,0 +1,44 @@
+dotnet-new
+==========
+
+## NAME
+dotnet-new -- Create a new .NET Core project
+
+## SYNOPSIS
+dotnet-new [--type] [--lang]
+
+## DESCRIPTION
+The new command provides a convenient way to initalize a valid .NET Core project and sample source code to try out the CLI toolset. 
+
+This command is invoked in the context of a directory. When invoked, the command will result in two main artifacts being dropped to the directory: 
```

Comment:
I dislike the term artifact. :smile: 
Can we just call them files in this context? 

When invoked, the command will produce two files in the current directory:
1.  A `Program.cs` ( or `Program.fs` ) file that contains a sample "Hello World" program.
...